runtime/lua/vim/lsp.lua

```diff
@@ -0,0 +1,147 @@
+-- luacheck: globals vim
+
+local lsp = {
+  server_config = require('vim.lsp.server_config'),
+  config = require('vim.lsp.config'),
+  protocol = require('vim.lsp.protocol'),
+  builtin_callbacks = require('vim.lsp.builtin_callbacks'),
+  checks = require('vim.lsp.checks'),
+  util = require('vim.lsp.util'),
+}
+
+local Client = require('vim.lsp.client')
+local callbacks = require('vim.lsp.callbacks')
+local server_config = require('vim.lsp.server_config')
+
+local log = require('vim.lsp.log')
```

Comment:
This should be ``vim.lsp.logger``, shoudln’t it?

Reply:
Thx :)